### ISSUE TITLE
Custom options for pip when environment_type is virtualenv

### ISSUE THREAD

**Issue by astrofrog** [spacetelescope/asv#436]
I would like to set up an asv environment using pip wheels that I built, so I would like to pass the path to the wheel repository to pip. Is there currently a way to tell asv extra options to include when calling pip?

**Comment by pv** [comment]
I think most pip options have corresponding environment variables that 
you can set.

**Comment by astrofrog** [comment]
@pv - ah yes, thanks, it looks like ``$PIP_FIND_LINKS`` should work. However, while it works when I just run pip, it doesn't seem to be working when asv creates environments. I tried turning on verbose mode for asv but I still don't see what's going on inside the environment creation. Is there a way to see the pip command and output?

**Comment by pv** [comment]
`asv -v` should print the commands and stdout (it does that for me, eg.
I can see the pip install command output once the command finishes).

**Comment by astrofrog** [comment]
@pv - in my case, the issue was that I am running on a raspberry pi and the pip command was incredibly slow, and timed out (since the compilation took so long). However, I wanted to be able to see the full pip debug output, and I eventually found the solution (it needed the wheel url to be https). I've solved my issue, so I think we can close this. It would be nice if there was a way to see the pip 'live' output easily, but we're all short on developer time, and I don't think that's very high priority (my use case was quite specialized)
